### PR TITLE
Feature/selenium add right click method

### PR DESCRIPTION
**Pull request by lstefaniszyn**

### CODE REVIEW

File: allure-app-under-test/src/test/java/com/capgemini/ntc/selenium/projectY/MouseClickTest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import com.capgemini.ntc.core.groupTestCases.testSuites.tags.TestFirefox;
+import com.capgemini.ntc.selenium.pages.projectY.ContextMenuPage;
+import com.capgemini.ntc.test.core.BaseTest;
+
+@Category(TestFirefox.class)
```
Comment (lstefaniszyn) [inline allure-app-under-test/src/test/java/com/capgemini/ntc/selenium/projectY/MouseClickTest.java]: please add here also "TestLocally"  tag

File: allure-framework-modules/allure-core-module/logs/main.log [hunk 2]
```diff
@@ -0,0 +1,7 @@
+2018-01-04 at 10:37:35 CET logInfo - [main] testMethod(com.capgemini.ntc.test.core.unit.TestWatcherExceptionTest$SetUpExceptionTest) STARTED.
```
Comment (lstefaniszyn) [inline allure-framework-modules/allure-core-module/logs/main.log]: please remove   /logs folder

File: allure-framework-modules/allure-core-module/logs/main_env.log [hunk 3]
```diff
@@ -0,0 +1 @@
+2018-01-04 at 10:37:35 CET logEnv - ----- test -----
```
Comment (lstefaniszyn) [inline allure-framework-modules/allure-core-module/logs/main_env.log]: please remove   /logs folder

**Comment by przwojtkow** [comment]
Updated!

**Pull request merged**